### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by DragonCK**
TEST

### CODE REVIEW

**Comment by DragonCK** [comment]
comment test

**Comment by OhMinsSup** [comment]
> TEST

A

> comment test
B

File: README.md [hunk 1]
```diff
@@ -99,7 +99,7 @@ SerializableError: Error serializing `.dehydratedState.queries[0].state.data.con
 
 https://qiita.com/jagaapple/items/faf125e28f8c2860269c
 
-## EP.11 write 페이지에서 에디터 적용하기 (6) - swr을 이용해서 ssr 및 csr 적용
+## EP.11 write 페이지에서 에디터 적용하기 (6) - swr을 이용해서 ssr 및 csr 적용.
```
Comment (OhMinsSup) [inline README.md:103]: ADD SINGLE COMMENT

Comment (DragonCK) [reply README.md:103]: ?????

Comment (DragonCK) [reply README.md:103]: START REVIEW

Comment (OhMinsSup) [reply README.md:103]: REVIEW COMMENT

**Review by OhMinsSup — COMMENTED** [review]
submit review comment

**Review by OhMinsSup — APPROVED** [review]
APPROVE Submit review

**Review by OhMinsSup — CHANGES_REQUESTED** [review]
REQUEST CHANGE

**Review by DragonCK — COMMENTED** [review]
COMMENT SUBMIT AAAAA

**Pull request merged**